estate/__manifest__.py

```diff
@@ -3,10 +3,13 @@
     'category': 'Real Estate',
     'depends': ['base_setup'],
     'application': True,
+    "installable": True,
```

Comment:
try to stick to the same text indicator. either " or ', but use the same everywhere :)